src/dbSchema.sql

```diff
@@ -32,7 +32,7 @@ CREATE TABLE history(idx int not null, chatid int64 not null, msgid int64 not nu
     userid int64, keyid int not null, type tinyint, updated smallint, ts int,
     is_encrypted tinyint, data blob, backrefid int64 not null, UNIQUE(chatid,msgid), UNIQUE(chatid,idx));
 
-CREATE TABLE sendkeys(chatid int64 not null, userid int64 not null, keyid int64 not null, key blob not null,
+CREATE TABLE sendkeys(chatid int64 not null, userid int64 not null, keyid int32 not null, key blob not null,
```

Comment:
Not sure if the migration of the DB to a newer version is really worthy

Reply:
Please check ticket where it's the link to full conversation about this topic.

Reply:
Please check ticket to see the conversation link about this topic.